### PR TITLE
Most of the updated XML documentation for the SQL cmdlets.

### PR DESCRIPTION
**Pull request by marceloyuela**
As the title suggests, this is most of the XML documentation for SQL cmdlets. The rest will be in a different pull request- I wanted to split it up so that it wasn't a ton of changes in one pull request.

### CODE REVIEW

File: Google.PowerShell/Sql/GcSqlFlagsCmdlet.cs [hunk 1]
```diff
@@ -25,8 +25,19 @@ namespace Google.PowerShell.Sql
...
+    ///   </para>
+    ///   <para><code>
+    ///     PS C:\> Get-GcSqlFlags
+    ///   </code></para>
+    ///   <br></br>
+    ///   <para>(If successful, the command returns a list of database flags available for instances)</para>
+    /// </example>
     /// </summary>
     [Cmdlet(VerbsCommon.Get, "GcSqlFlags")]
+    [OutputType(typeof(IList<Flag>))]
```
Comment (ILMTitan) [inline Google.PowerShell/Sql/GcSqlFlagsCmdlet.cs]: I think this should just be typeof(Flag). We give WriteObject an IList<Flag>, but then we tell it to iterate through the elements.

File: Google.PowerShell/Sql/GcSqlInstanceCmdlets.cs [hunk 2]
```diff
@@ -21,6 +21,8 @@ namespace Google.PowerShell.Sql
     /// </para>
     /// </summary>
     [Cmdlet(VerbsCommon.Get, "GcSqlInstance", DefaultParameterSetName = ParameterSetNames.GetList)]
+    [OutputType(typeof(DatabaseInstance))]
+    [OutputType(typeof(IEnumerable<DatabaseInstance>))]
```
Comment (ILMTitan) [inline Google.PowerShell/Sql/GcSqlInstanceCmdlets.cs]: Again, this should probably only be DatabaseInstance.

File: Google.PowerShell/Sql/GcSqlOperationCmdlet.cs [hunk 3]
```diff
@@ -19,8 +19,30 @@ namespace Google.PowerShell.Sql
...
+    ///   </para>
+    ///   <para><code>
+    ///     PS C:\> Get-GcSqlOperation -Name "1d402..."
+    ///   </code></para>
+    ///   <br></br>
+    ///   <para>(If successful, the command returns an Operation resource for the specified operation.)</para>
+    /// </example>
     /// </summary>
     [Cmdlet(VerbsCommon.Get, "GcSqlOperation")]
+    [OutputType(typeof(IEnumerable<Operation>))]
```
Comment (ILMTitan) [inline Google.PowerShell/Sql/GcSqlOperationCmdlet.cs]: Delete

File: Google.PowerShell/Sql/GcSqlSslCmdlets.cs [hunk 4]
```diff
@@ -19,8 +19,40 @@ namespace Google.PowerShell.Sql
...
+    ///   <para><code>
+    ///     PS C:\> Get-GcSqlSslCert "myInstance" "myFinger"
+    ///   </code></para>
+    ///   <br></br>
+    ///   <para>(If successful, the command return a single SSL Certificate.)</para>
+    /// </example>
     /// </summary>
     [Cmdlet(VerbsCommon.Get, "GcSqlSslCert")]
+    [OutputType(typeof(SslCert))]
+    [OutputType(typeof(IList<SslCert>))]
```
Comment (ILMTitan) [inline Google.PowerShell/Sql/GcSqlSslCmdlets.cs]: Delete

File: Google.PowerShell/Sql/GcSqlTiersCmdlet.cs [hunk 5]
```diff
@@ -21,8 +21,19 @@ namespace Google.PowerShell.Sql
...
+    ///   </para>
+    ///   <para><code>
+    ///     PS C:\> Get-GcSqlTiers
+    ///   </code></para>
+    ///   <br></br>
+    ///   <para>(If successful, the command returns a list of tiers available for your project.)</para>
+    /// </example>
     /// </summary>
     [Cmdlet(VerbsCommon.Get, "GcSqlTiers")]
+    [OutputType(typeof(IList<Tier>))]
```
Comment (ILMTitan) [inline Google.PowerShell/Sql/GcSqlTiersCmdlet.cs]: Should just be Tier.

**Comment by ILMTitan** [comment]
WriteOutput(collection, true); is essentially the same as foreach(var o in collection) { WriteOutput(o);}. The output type is the type of the collection members. Even if what we get back is an array of them, it is still useful to be able to intellisense their properties directly.

**Comment by marceloyuela** [comment]
Fixed the above requests

**Comment by ILMTitan** [comment]
LGTM

**Pull request merged**